### ISSUE TITLE
Gutenboarding: Accessibility issues with new Acquire Intent design.

### ISSUE THREAD

**Issue by yansern** [Automattic/wp-calypso#40013]
- [x] 1. If the vertical text is long, focusing in & out of the input box clips the text. Fixed in #40645
![2020-03-09_20-58-21 (1)](https://user-images.githubusercontent.com/1287077/76215400-a47cd300-6249-11ea-92c0-d407634b6158.gif)

- [x] 2. When focused on site name input, unable to <kbd>Shift</kbd> + <kbd>Tab</kbd> back to vertical text input.
Fixed in #40645

- [ ] 3. Automatically remove trailing . if user entered them? I subsconciously added a . because there was not one when the input is being focused. See gif:
![2020-03-09_20-59-03 (1)](https://user-images.githubusercontent.com/1287077/76215827-7ea3fe00-624a-11ea-849b-293c1c620400.gif)

- [ ] 4. Missing focus outline when tabbing on the "Choose a design" button.

- [x] 5. Weird wrapping behaviour, Maybe using `contenteditable` instead of `input` might help with wrapping but that will introduce a whole bunch of other technical challenges. 
Fixed in #40645
![image](https://user-images.githubusercontent.com/1287077/76216002-cb87d480-624a-11ea-9203-636efdb02938.png)

**Comment by yansern** [comment]
Closing this and continuing with this. https://github.com/Automattic/wp-calypso/issues/49419
